BUILD

```diff
@@ -0,0 +1,73 @@
+# Description:
+# Motion interchange format.
+
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+objc_library(
+    name = "MotionInterchange",
+    srcs = glob([
+        "src/*.m",
+        "src/private/*.m",
+    ]),
+    hdrs = glob([
+        "src/*.h",
+        "src/private/*.h",
+    ]),
+    enable_modules = 1,
+    includes = ["src"],
+    visibility = ["//visibility:public"],
+    copts = [
+        "-Wall",  # Standard known-to-be-bugs warnings.
+        "-Wcast-align",  # Casting a pointer such that alignment is broken.
+        "-Wconversion",  # Numeric conversion warnings.
+        "-Wdocumentation",  # Documentation checks.
+        "-Werror",  # All warnings as errors.
+        "-Wextra",  # Many useful extra warnings.
+        "-Wimplicit-atomic-properties",  # Dynamic properties should be non-atomic.
+        "-Wmissing-prototypes",  # Global function is defined without a previous prototype.
+        "-Wno-error=deprecated",  # Deprecation warnings are never errors.
+        "-Wno-error=deprecated-implementations",  # Deprecation warnings are never errors.
+        "-Wno-sign-conversion",  # Do not warn on sign conversions.
+        "-Wno-unused-parameter",  # Do not warn on unused parameters.
+        "-Woverlength-strings",  # Strings longer than the C maximum.
+        "-Wshadow",  # Local variable shadows another variable, parameter, etc.
+        "-Wstrict-selector-match",  # Compiler can't figure out the right selector.
+        "-Wundeclared-selector",  # Compiler doesn't see a selector.
+        "-Wunreachable-code",  # Code will never be reached.
```

Comment:
Wishful thinking: It would be nice if these were defined once, across all of our projects.
Consider seeing if there are additional useful flags in MDC-iOS:
https://github.com/material-components/material-components-ios/blob/develop/catalog/MaterialComponentsWarnings.xcconfig

Reply:
PR is already awaiting your approval :P https://github.com/material-foundation/bazel_ios_warnings/pull/1